validphys2/src/validphys/lhapdfset.py

```diff
@@ -28,9 +28,10 @@
 """
 import logging
 
-import lhapdf
 import numpy as np
 
+from validphys.lhapdf_compatibility import make_pdf
```

Comment:
```suggestion
from validphys.lhapdf_compatibility as lhac
```
The more obvious `lc`  is already an existing variable in some places.

I'd prefer to keep the namespace a bit cleaner (especially if we import functions with such generic names as `make_pdf`).